src/index.js

```diff
@@ -25,18 +26,20 @@ function main(client, options, _performance) {
 		}
 	}
 
-	const invokeFunctions = function () {
+	const invokeFunctions = function(subset) {
 		const evolv = window.evolv;
 		if (typeof evolv === 'undefined' || !evolv.javascript || !evolv.javascript.variants) {
 			if (!applyTimeout) {
 				return;
 			}
-			
+
 			const timeNow = Date.now();
 			const domContentLoadedEventStart = (_performance || performance).timing.domContentLoadedEventStart;
 			var threshold = options.timeoutThreshold || 60000;
 			if (domContentLoadedEventStart === 0 || timeNow < domContentLoadedEventStart + threshold) {
-				setTimeout(invokeFunctions, MAX_TIMEOUT);
+				setTimeout(function() {
```

Comment:
Did you change this? I though we werent passing the subset for this?

Reply:
I thought you chose not to give the subset to the timeout

Reply:
I did change it after a discussion with Rob. We opted to let all of the timeouts get their chance. One of the changes we made later on ended up handling the behavior anyway.

Reply:
Oh ok